### PR TITLE
Fix get modules paginated

### PR DESCRIPTION
**Pull request by cristovaoth**
Fixes [112](https://github.com/gnosis/zodiac/issues/112)

### CODE REVIEW

File: test/03_Modifier.spec.ts [hunk 1]
```diff
@@ -174,21 +175,49 @@ describe("Modifier", async () => {
...
+
+      expect(
+        await modifier.getModulesPaginated(user1.address, 1)
+      ).to.be.deep.equal([[], AddressOne]);
+
+      await expect(
+        modifier.getModulesPaginated(user2.address, 1)
+      ).to.be.revertedWith(`InvalidModule`);
+    });
+    it("returns empty array if no modules are enabled.", async () => {
```
Comment (auryn-macmillan) [inline test/03_Modifier.spec.ts:199]: Looks like this test is duplicated immediately below.

Comment (cristovaoth) [reply test/03_Modifier.spec.ts:203]: Fixed

**Comment by auryn-macmillan** [comment]
@cristovaoth, I was going to merge this, but it looks like some tests are failing.

**Review by auryn-macmillan — APPROVED** [review]
lgtm

**Pull request merged**